connector_pageviews.config

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "datagen-pageviews",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "pageviews",
+    "quickstart": "pageviews",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
```

Comment:
Unless the connector requires these converters in order to work (and I don't think it does), we can probably leave the `key.converter` and `value.converter` configs out of these sample configs.

Reply:
We need them for Confluent CLI to override the default which is the AvroConverter

Reply:
Does the data generated by the connector not work with the `AvroConverter`?

Reply:
The connector does work with Avro, but the quickstarts are currently not using Avro

Reply:
Gotcha, thanks for clarifying.